tests/test_schemeless.py

```diff
@@ -70,6 +70,26 @@ def test_creation(self) -> None:
         with self.assertRaises(ValueError):
             ResourcePath(relative, root=ResourcePath("resource://lsst.resources/something.txt"))
 
+    def test_isdir(self):
```

Comment:
Unless I'm way off base on my comments above, I think we may be missing some test coverage for the other ResourcePath methods in the schemeless forceDirectory=None case.

Reply:
The full coverage report indicates that we are in the high 90s% for schemeless and resource path. I can't see any obvious gaps from the report.